rspec-tools/tests/resources/rules/S100/metadata.json

```
@@ -39,5 +39,11 @@
     "defense_in_depth",
     "never_trust_user_input"
   ],
-  "quickfix":"unknown"
+  "quickfix":"unknown",
```

```suggestion
  "quickfix": "unknown",
```

Not related to your PR but if we change the line anyway.